ui/src/app/components/genomics-extraction-table.tsx

```diff
@@ -230,17 +237,19 @@ const FailedRequestMessage = () => <div style={{textAlign: 'center'}}>
   </FlexRow>
 </div>;
 
-export const GenomicsExtractionTable = fp.flow(withCurrentWorkspace())(({workspace}) => {
-  const [extractionJobs, setExtractionJobs] = useState(null);
+export const GenomicsExtractionTable = fp.flow(
+  withCurrentWorkspace(),
+  withStore(genomicExtractionStore, 'genomicExtraction')
+)(({workspace, genomicExtraction}) => {
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
     dataSetApi().getGenomicExtractionJobs(workspace.namespace, workspace.id)
-      .then(resp => setExtractionJobs(resp.jobs))
+      .then(resp => updateGenomicExtractionStore(workspace.namespace, resp.jobs))
       .finally(() => setIsLoading(false));
   }, [workspace]);
 
-  const requestFailed = !isLoading && extractionJobs === null;
+  const requestFailed = !isLoading && genomicExtractionStore.get() === null;
```

Comment:
actually, it should be `genomicExtraction[workspace.namespace]` right? My bad, I wasn't sure what the API for the store was in my original comment.

Also, wdyt of just creating `const extractionJobs = genomicExtraction[workspace.namespace]`? We call it a few times below.

Reply:
I think what I actually want is to check that thing for `undefined`, which is what you get when you try to access a key in an associative array that doesn't exist. whereas empty array is a valid value.